ush/generate_FV3LAM_wflow.py

```diff
@@ -137,9 +140,23 @@ def generate_FV3LAM_wflow(
         verbose=debug,
     )
 
-    create_symlink_to_file(
-        wflow_launch_script_fp, os.path.join(exptdir, wflow_launch_script_fn), False
-    )
+    with open(wflow_launch_script_fp, "r") as launch_script_file:
+        launch_script_content = launch_script_file.read()
+
+    # Stage an experiment-specific launch file in the experiment directory
+    template = Template(launch_script_content)
+
+    # The script needs several variables from the workflow and user sections
+    template_variables = {**expt_config["user"], **expt_config["workflow"],
+            "valid_vals_BOOLEAN": list_to_str(expt_config["constants"]["valid_vals_BOOLEAN"])}
+    launch_content =  template.safe_substitute(template_variables)
+
+    launch_fp = os.path.join(exptdir, wflow_launch_script_fn)
+    with open(launch_fp, "w") as expt_launch_fn:
+        expt_launch_fn.write(launch_content)
+
+    os.chmod(launch_fp, os.stat(launch_fp).st_mode|stat.S_IXUSR)
+
```

Comment:
@mkavulich To fix the issue you brought up, I decided it might be best to go ahead and "configure" this launch script, just like all the other files in the experiment directory. Here is the additional code that works, and there were a couple of items in the launch script itself that needed to change to make it all work.

The idea is that we have a chicken and egg problem. It can't "source" the YAML file before it knows more information about the `USHdir` and the platform so it can load the appropriate module files. Since we know these at configuration time, I chose to fill them in and place the now hard-coded (AKA preconfigured) launch script in the experiment directory. It won't need to do any sourcing of the var_defns file to complete its task.